views/js/qtiCreator/widgets/choices/helpers/formElement.js

```diff
@@ -23,6 +23,16 @@ define(['jquery'], function ($) {
                 interaction = choice.getInteraction(),
                 $shuffleToggle = $container.find('[data-role="shuffle-pin"]');
 
+            const pinIcon = function (icon) {
```

Comment:
better use name `$icon` it is our code style -use $ when it is jQuery element

Reply:
done